index.js

```diff
@@ -162,6 +164,58 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.split(" ").splice(1).join(" ");
+  const fuzzySearchUrl =
+    "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=";
+
+  console.log(fuzzySearchUrl + fuzzySearchKey);
```

Comment:
you can just include `fuzzySearchKey` in `fuzzySearchUrl` (e.g. `const fuzzySearchUrl = "http://etc" + fuzzySearchKey`)